src/room/track/RemoteTrackPublication.ts

```diff
@@ -45,15 +51,19 @@ export default class RemoteTrackPublication extends TrackPublication {
       ],
     };
     this.emit(TrackEvent.UpdateSubscription, sub);
+    this.emitSubscriptionUpdateIfChanged(prevStatus);
   }
 
   get subscriptionStatus(): TrackPublication.SubscriptionStatus {
-    if (this.subscribed === false || !super.isSubscribed) {
-      if (!this.allowed) {
-        return TrackPublication.SubscriptionStatus.NotAllowed;
-      }
+    if (!this.allowed) {
+      return TrackPublication.SubscriptionStatus.NotAllowed;
+    }
+    if (!this.subscribed) {
```

Comment:
would this be better? since initial value inherits from autoSubscribe, which is true by default

```suggestion
    if (this.subscribed === false) {
```